documentation/service-rules/school-admins.md

```diff
@@ -117,11 +133,243 @@ We ask this question so we can get a better understanding of when the ECT is sta
+
+So, if a school user changes the TRN, it would mean the value it's changed to might not validly be an ECT that should be registered for training. It could also change it to a different person with a different name . Therefore, we don't allow changes to the TRN. If they want to change the TRN, they'd have to start the whole registration process again.
+
+The school start date provided might impact what lead provider information a school user selects.
+
+If a school has registered an ECT using previously used programme details, if they decide to change the appropriate body, lead provider, delivery partner or programme type, we'd also just take them back to the question if they want to use previously used programme details again. This simplifies the build and also makes sure they don't pick any options that might not be feasible.
+
+Once a school confirms the details, it saves the details for that ECT. The ECT record is created, with a status of 'mentor required'. It creates an ECT record that can be viewed on the 'ECTs' page. However, registration is still not finished!
+
+## Assigning a mentor to an ECT
+
+### Assigning a mentor immediately after giving ECT details
+
+Registration is only finished when an ECT is also given a mentor.
+
+All ECTs need mentors. Whilst they can be eligible for training before this, usually missing mentors means mentors don't undergo mentor training as they should, and it also means any mentors not assigned to an ECT won't receive funding.
+
+Schools can choose to assign a mentor for that ECT immediately after saving the details for the ECT, or they can come back later and do this through the homepage. You can read more about why we chose to do this [in this design history here](https://teacher-cpd.design-history.education.gov.uk/ecf-v2/assigning-mentors/).
+
+If they do it immediately, they're taken to the journey to assign a mentor.
+
+More about the process for assigning a mentor is given in the sections below, but we still don't see ECTs as fully registered until they have a mentor.
+
+### Assigning a mentor through the ECTs page
+
+Mentors can also be assigned later, if the school doesn't have the information or time to do it immediately after registering an ECT.
+
+Currently, a school user would only be able to do this by going to the 'ECTs' page.
+
+There would be a prompt next to every ECT that has a saved record to assign a mentor, if they don't already have one.
+
+If the school user selects to assign the ECT a mentor, they're taken into the journey to assign a mentor.
+
+### Deciding if the mentor is new or a previously registered mentor
+
+Regardless of where the journey to assign a mentor starts, schools are taken to a question where they are asked if they're assigning a mentor they've already registered, or registering a new mentor.
+
+This will show all existing mentors that are still actively at the school - they have a open mentor_at_school period.
```

Comment:
```suggestion
This will show all existing mentors that are still actively at the school - they have a open `mentor_at_school_period`.
```

Reply:
Fixed in: https://github.com/DFE-Digital/register-early-career-teachers-public/pull/325/commits/5bb7b5b433f25f782d9748df41b67d17cf49147d